src/theme/foundations/colours.ts

```diff
@@ -1,4 +1,26 @@
 // Colour schemes available for isomer
+
+export type IsomerBaseColorScheme = "purple"
+
+// We are not using all the shades of the base colour scheme,
+// but this is placed here if we need them in the future.
+export const baseColourMapping: {
```

Comment:
could we rename this to `palettes`? more indicative of what it's supposed to be

Reply:
646f8aaf4d994084f96998cd864ef963623aea02